### PR TITLE
[Community Tutuorials] Preparing Beginner's Cookbook for publication

### PR DESCRIPTION
**Pull request by gino-m**

### CODE REVIEW

**Comment by gino-m** [comment]
@jdbcode could you kindly take a look at these tweaks for publication on our Developers Docs?

@TC25 I've made some minor edits; please take a look and let me know if there's anything you object to!

**Comment by TC25** [comment]
> @jdbcode could you kindly take a look at these tweaks for publication on our Developers Docs?
> 
> @TC25 I've made some minor edits; please take a look and let me know if there's anything you object to!

@gino-m I think the changes look great. Thanks!

**Comment by jdbcode** [comment]
> A series of minor edits. @TC25 will you please make the changes.
>
> A general note: ee.Filter.metadata is deprecated - to promote best practices can you use .filterMetadata or .filter(ee.Filter.gte()) (w/ appropriate condition) where ee.Filter.metadata is used.
>
>This is great! It's really close.

@TC25 Disregard the previous message. I'll make these edits.

**Comment by TC25** [comment]
> > A series of minor edits. @TC25 will you please make the changes.
> > A general note: ee.Filter.metadata is deprecated - to promote best practices can you use .filterMetadata or .filter(ee.Filter.gte()) (w/ appropriate condition) where ee.Filter.metadata is used.
> > This is great! It's really close.
> 
> @TC25 Disregard the previous message. I'll make these edits.

Thanks @jdbcode . I think the pull request was already merged from my end. I was wondering how to edit this version (would have had to fork again?).

**Comment by gino-m** [comment]
> Thanks @jdbcode . I think the pull request was already merged from my end. I was wondering how to edit this version (would have had to fork again?).

@jdbcode and I are making the requested edits, so you shouldn't need to do it yourself. But if there's something you do want to change on your own, you should be able to edit the files in this PR directly via the GitHub web UI. If you do, please let me know if it works.

**Comment by jdbcode** [comment]
>@jdbcode and I are making the requested edits, so you shouldn't need to do it yourself. But if there's something you do want to change on your own, you should be able to edit the files in this PR directly via the GitHub web UI. If you do, please let me know if it works.

@TC25 please hold off on any changes you might want to make. I'm currently editing the .md file and don't want to create any conflicts.

File: tutorials/beginners-cookbook/index.md [hunk 1]
```diff
@@ -1,427 +1,463 @@
...
+What is Earth Engine?
+
+- A cloud-based platform for planetary scale geospatial analysis
 - Uses Google's computational resources to reduce processing time
-- Massive archive of remote sensing data 
-  - 200+ public datasets 
-  - over 4,000 new images every day 
-  - over 5 million images 
+- A massive archive of remote sensing data
+  - 200+ public datasets
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 2]
```diff
@@ -1,427 +1,463 @@
...
 ### Adding a layer to the map
+
 ```javascript
 Map.addLayer(variableName);
 ```
 
----
-
-## Variable types in Earth Engine
+## Earth Engine data types
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done - used "Common"

File: tutorials/beginners-cookbook/index.md [hunk 3]
```diff
@@ -1,427 +1,463 @@
...
 ```javascript
 var result = input.map(functionName);
 ```
-Mapping a function over a collection applies the function to every element in the collection.
-
----
 
+Mapping a function over a collection applies the function to every element in the collection.
 
 ## Operations on geometries
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done - use "Common"

File: tutorials/beginners-cookbook/index.md [hunk 4]
```diff
@@ -1,427 +1,463 @@
...
-### Create buffer around geometry
+
+### Creating buffer around a geometry
+
 ```javascript
 var buffGeo = geometry.buffer(100);
 ```
-### Find bounded rectangle of the geometry
+
+### Finding thing bounding rectangle of the geometry
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 5]
```diff
@@ -1,427 +1,463 @@
...
+
 ```javascript
 var bounGeo = geometry.bounds();
 ```
-### Find the smallest envelope that can envelop the geometry
+
+### Finding the smallest polygon that can envelop the geometry
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 6]
```diff
@@ -1,427 +1,463 @@
...
-// Get the union of all the county geometries in Connecticut
+// Get the union of all the county geometries in Connecticut.
 var countyConnectDiss=countyConnect.union();
-// Add the layer to the map with a specified color and layer name
+// Add the layer to the map with a specified color and layer name.
 Map.addLayer(countyConnectDiss, {color: 'red'}, 'Chicago dissolved');
-// Find the rectangle that emcompasses the southernmost, westernmost, easternmost, and northernmost
-// points of the feature
+// Find the rectangle that emcompasses the southernmost, westernmost,
+// easternmost, and northernmost points of the feature
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 7]
```diff
@@ -1,427 +1,463 @@
...
-// Add the layer to the map with a specified color and layer name
+// Add the layer to the map with a specified color and layer name.
 Map.addLayer(union, {color: 'purple'}, 'Bound and convex union');
 // Find the difference between two geometries
 var diff=bound.difference(convex, 100);
-// Add the layer to the map with a specified color and layer name
+// Add the layer to the map with a specified color and layer name.
 Map.addLayer(diff, {color: 'purple'}, 'Bound and convex difference');
-// Find area of feature
+// Find area of feature,
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 8]
```diff
@@ -1,427 +1,463 @@
...
 
-### Create a filter for particular days of the year
+### Filtering by particular days of the year
 
 ```javascript
 var dayFilter = ee.Filter.dayOfYear(startDay, stopDay);
 ```
 
-### Create a filter to subset geospatial data
+### Filteing by a bounding area
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 9]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
 ```
----
-## Operations on Images
+
+## Operations on images
 
 ### Selecting the bands of an image
+
 ```javascript
 var band = var_Image.select(bandName);
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 10]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
 ```
->or .neq or .gt or .gte or .lt or .lte
 
-### Applying masks
+> or .neq or .gt or .gte or .lt or .lte
+
+### Applying image masks
+
 ```javascript
 var masked = image.mask(mask);
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 11]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
 
-### Select images within geometry
+### Selecting images within a bounding geometry
+
 ```javascript
 var selectedIm = imCollection.filterBounds(geometry);
 ```
 
-### Perform pixelwise calculations for all images in collection
+### Performing pixelwise calculations for all images in acollection
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 12]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
+
 ```javascript
 newImage = oldImage.leftShift(valueOfShift);
 ```
->or .rightShift
 
-### Create a single value from an image by applying a reducer based on regions of interest
+> or .rightShift
+
+### Reducing an image down to a single value for particular area of interest
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 13]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
-var selectedIm = imCollection.filter(ee.Filter.metadata(propertyName, relation, value));
+var selectedIm = imCollection.filter(
+  ee.Filter.metadata(propertyName, relation, value));
 ```
 
->Relations could be "equals", "less_than", "greater_than", "starts_with", "ends_with", and "contains"
+> Relations could be `equals`, `less_than`, `greater_than`, `starts_with`,
+> `ends_with`, and `contains`.
+
+### Selecting images within date range
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 14]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
 ```javascript
 var sumOfImages = imCollection.reduce(ee.Reducer.first());
 ```
-#### Image and image collections operations example
-Let's analyze images over a region of interest (the counties of Connecticut). Images and image collections cover the bulk of Earth Engine's data archives.
+
+#### Example: Image and image collection operations
+
+Let's analyze images over a region of interest (the counties of Connecticut).
+Images and image collections cover the bulk of Earth Engine's data archives.
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 15]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
-
+### Exporting a collection to Google Drive, Earth Engine Asset, or Google Cloud
 ```javascript
 Export.image.toDrive({
   collection: varImage, description: 'fileName', region: geometry, scale: 1000
 });
 ```
->or image.toCloudStorage, image.toAsset, table.toDrive, table.toCloudStorage, video.toCloudStorage, and video.toDrive
+> or `Export.image.toCloudStorage()`, `Export.image.toAsset()`,
+> `Export.table.toDrive()`, `Export..table.toCloudStorage()`,
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 16]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
- });
+  // Load US county dataset.
+  var countyData = ee.FeatureCollection('TIGER/2018/Counties');
+  // Filter the counties that are in Connecticut.
+  // This will be the region of interest for the operations.
+  var roi=countyData.filter(ee.Filter.metadata('STATEFP', 'equals', '09'));
+  return image.reduceRegions({
+    // Collection to run operation over.
+    collection: roi,
+    // Take mean of all pixels in region.
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

File: tutorials/beginners-cookbook/index.md [hunk 17]
```diff
@@ -660,30 +718,18 @@ Export.video.toDrive({
...
 - [Global Surface UHI Explorer](https://yceo.users.earthengine.app/view/uhimap)
-
 - [Stratifi - cloud-based stratification](https://sabrinaszeto.users.earthengine.app/view/stratifi)
----
-
-## Resources
 
+## Additional resources
 
 - [Geospatial Software Design](https://environment.yale.edu/courses/2018-2019/detail/754/)
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Fixed

File: tutorials/beginners-cookbook/index.md [hunk 18]
```diff
@@ -433,225 +469,247 @@ var newFilterAnd = ee.Filter.and(listOfFilters);
...
-// Calculate mean of all images (pixel-by-pixel) in the collection
+// Calculate mean of all images (pixel-by-pixel) in the collection.
 var mean = bandSel1.mean();
-// Isolate image to region of interest
+// Isolate image to region of interest.
 var clipped = mean.clip(roi);
 // mathematical operation on image pixels to convert from digital number
-// of satellite observations to degree Celsius
+// of satellite observations to degree Celsius.
 var calculate = clipped.multiply(.02).subtract(273.15);
```
[the remark this answers is not in the document]
Comment (jdbcode) [reply tutorials/beginners-cookbook/index.md]: Done

**Comment by jdbcode** [comment]
@gino-m I made minor edits to your PR. Maybe just do a quick review of my changes and then merge.

**Comment by gino-m** [comment]
@jdbcode Sorry for the typos in my first pass, thanks a lot for all the fixes!

**Pull request merged**